ci-operator/step-registry/telcov10n/slcm/slcm-lab/telcov10n-slcm-slcm-lab-commands.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+# Fix user IDs in a container
+~/fix_uid.sh
+
+BASTION_ADDRESS="$(cat /var/run/bastion1/bastion-address)"
+VPN_URL="$(cat /var/run/bastion1/vpn-url)"
+VPN_USERNAME="$(cat /var/run/bastion1/vpn-username)"
+# For password with special characters
+VPN_PASSWORD=`cat /var/run/bastion1/vpn-password`
+SSH_KEY_PATH=/var/run/ssh-key/ssh-key
+SSH_KEY=~/key
+JUMP_SERVER_ADDRESS="$(cat /var/run/bastion1/jump-server)"
+IFNAME=tun10
+
+cp $SSH_KEY_PATH $SSH_KEY
+chmod 600 $SSH_KEY
+
+SSHOPTS=(-o 'ConnectTimeout=5'
+  -o 'StrictHostKeyChecking=no'
+  -o 'UserKnownHostsFile=/dev/null'
+  -o 'ServerAliveInterval=90'
+  -o LogLevel=ERROR
+  -i "${SSH_KEY}")
+
+# Run commands from the bastion
+timeout -s 9 10m ssh "${SSHOPTS[@]}" "kni@${JUMP_SERVER_ADDRESS}" bash -s -- \
+  "'${VPN_URL}'" "'${VPN_USERNAME}'" "'${VPN_PASSWORD}'" "'${IFNAME}'" "'${BASTION_ADDRESS}'" << 'EOF'
+    set -o nounset
```

Comment:
Is this a duplication of line 3?

Reply:
Line 3 runs in the jump server in lab1 while this script runs in lab2

Reply:
Ah OK.